src/styles/blocks/main.scss

```diff
@@ -0,0 +1,32 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  justify-items: center;
+}
+
+@media (max-width: 487px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
```

Comment:
The media query at line 10 is not necessary. It sets the grid-template-columns property to repeat(1, 200px), but this is the default behaviour when the viewport width is less than the minimum width of a column. You can safely remove this media query.